app/serializers/preprint.ts

```diff
@@ -1,6 +1,19 @@
+import Model from '@ember-data/model';
+import { Resource } from 'osf-api';
+
 import OsfSerializer from './osf-serializer';
 
 export default class PreprintSerializer extends OsfSerializer {
+    normalize(modelClass: Model, resourceHash: Resource) {
+        const result = super.normalize(modelClass, resourceHash);
+        // Insert a `versions` relationship to the model
+        result.data.relationships!.versions = {
+            links: {
+                related: resourceHash.links!.preprint_versions!,
+            },
+        };
```

Comment:
Why is this necessary?

Reply:
According to the [API Contract](https://docs.google.com/document/d/1-3mWJzH8t6XD5RYzKhTSo1dm5wyIwle4IEzSzVkqVcc/edit?tab=t.0#heading=h.599xafv6pi9c), the API won't be serializing this relationship, since it is recursive.